src/Area.ts

```diff
@@ -91,7 +91,7 @@ export class Area extends GameEntity {
 	 * @param {string} id Room id
 	 * @return {Room|undefined}
 	 */
-	getRoomById(id: string) {
+	getRoomById(id: string): Room | never {
```

Comment:
To me, `never` indicates that this will throw an error rather than return anything besides `Room` (which is true, this should not return undefined...)

Reply:
I haven't used that method at all in my repo, typically use the RoomManager.getRoom, which is similar.